test/test_utils.py

```diff
@@ -130,20 +116,99 @@ def test_get_ip_without_internet(self, mock_get):
 class TestAssertConfigsEquivalent(unittest.TestCase):
     def test_same_configs(self):
         self.assertTrue(
-            assert_configs_are_equivalent_besides_ids(XRAY_CONFIG, XRAY_CONFIG)
+            assert_configs_are_equivalent_besides_ids(
+                copy.deepcopy(XRAY_CONFIG), copy.deepcopy(XRAY_CONFIG)
```

Comment:
I don't think this copy is necessary since the util performs a copy.

Reply:
Yeah I'll remove these copy.deepcopies